### PR TITLE
updating axios version

### PR DESCRIPTION
**Pull request by Erli-ms**
# **Thank you for your contribution. Before submitting this PR, please include:**

### Description

Include a description of the problem to be solved.

## Solution Proposed

Detail what is the solution proposed, include links to design document if required or any other document required to support the solution.

### Acceptance criteria

Define what are the conditions to consider the PR has achieved the intended goal

## Test cases and evidence

Include what test cases were considered, any evidence of testing for future references, to identify any corner cases, etc.

### Sanity Tests

- [ ] You have tested all changes in Popout mode
- [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
- [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y

- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

***Please provide justification if any of the validations has been skipped.**_

### CODE REVIEW

**Review by elopezanaya — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [0.5.21]
+
+### Changed
+
+- Update axios version from ^1.8.2 to ^1.12.0
```
Comment (elopezanaya) [inline CHANGELOG.md:11]: this is the wrong format,  follow existent pattern for uptakes

**Comment by elopezanaya** [comment]
follow format  as > 
- Uptake [axios@1.7.4](https://www.npmjs.com/package/axios/v/1.7.4)
